src/client/initClient.jsx

```diff
@@ -46,6 +48,8 @@ export default async function initClient() {
       return;
     }
 
+    await loadPWA();
```

Comment:
does loading the Service Worker need to block the app from rendering?

Reply:
You are correct, it doesn't need to block the app from rendering... it would be beneficial to set this in motion as early as possible but I will remove awaiting it to allow the app to render.

Reply:
Updated to allow it to load at its own pace and not block.